src/gt4py/backend/python_generator.py

```diff
@@ -155,6 +177,11 @@ def visit_BinOpExpr(self, node: gt_ir.BinOpExpr):
 
         return source
 
+    def visit_NativeFuncCall(self, node: gt_ir.NativeFuncCall):
+        call = self.NATIVE_FUNC_TO_PYTHON[node.func]
+        args = ",".join([self.visit(arg) for arg in node.args])
```

Comment:
Any iterable works here, it is not needed to explictly create a new list
```suggestion
        args = ",".join(self.visit(arg) for arg in node.args)
```

Reply:
done